src/binding.cc

```diff
@@ -682,6 +682,26 @@ v8::PrimitiveArray* v8__ScriptOrModule__GetHostDefinedOptions(
   return local_to_ptr(self.GetHostDefinedOptions());
 }
 
+v8::SharedArrayBuffer* v8__SharedArrayBuffer__New(v8::Isolate* isolate, size_t byte_length) {
+  return local_to_ptr(v8::SharedArrayBuffer::New(isolate, byte_length));
+}
+
+size_t v8__SharedArrayBuffer__ByteLength(v8::SharedArrayBuffer& self) {
+  return self.ByteLength();
+}
+
+void v8__SharedArrayBuffer__GetContents(v8::SharedArrayBuffer& self, v8::SharedArrayBuffer::Contents* out) {
+  *out = self.GetContents();
+}
```

Comment:
SharedArrayBuffer::GetContents is deprecated:
https://cs.chromium.org/chromium/src/v8/include/v8.h?l=5698&rcl=4c53f9a51444393133ff303952f1296603d44ab7

Reply:
Didn't see that will refactor to use `BackingStore`.